### DISCUSSION TITLE
firefox: How to run from ramdisk?

### DISCUSSION THREAD

**Discussion by WJFV** [netblue30/firejail#4250] [Q&A] [answered]
Hallo guys,

first of all, thank you very much for your great work with firejail.

I hope I am right here with my concern.

I have my Firefox profile and cache folder on the ramdisk:

/media/ramdisk/.mozilla
/media/ramdisk/.cache/mozilla

I have linked the correlating directories in the home folder to these directories in the ramdisk using "ln".
Now I want to start Firefox with Firejail and create a profile file for this, but I can't get any further.

How do I have to create a corresponding profile file? What should this profile file look like? Many Thanks!

**Comment by rusty-snake** [comment, marked as answer]
It's probably enough to allow firefox to access `/media` by `ignore disable-mnt`. However a bit more hardening would be nice.
Add this to your firefox.local:
```
ignore disable-mnt
blacklist /mnt
blacklist /run/mnt
blacklist /run/media
whitelist /media/ramdisk/.mozilla
whitelist /media/ramdisk/.cache/mozilla
```

**Reply by WJFV** [reply]
That worked great for me, thank you!
